backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/databases/postgres/repositories/MoviePostgresRepository.kt

```diff
@@ -0,0 +1,115 @@
+package com.pk.zbtz.zbtzbackend.databases.postgres.repositories
+
+import com.pk.zbtz.zbtzbackend.databases.postgres.models.MoviePostgresModel
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
+import org.springframework.stereotype.Repository
+import java.time.Year
+
+@Repository
+interface MoviePostgresRepository: JpaRepository<MoviePostgresModel, Int> {
+    @Query(
+        value = "select * from movies offset (:offset)",
+        nativeQuery = true
+    )
+    fun findAllWithOffset(
+        @Param("offset") offset: Int
+    ): List<MoviePostgresModel>
```

Comment:
Let's change the calculating pagination to be the same as in Mongo because the frontend doesn't work well for now. I recommend using [PageRequest](https://www.baeldung.com/spring-data-jpa-pagination-sorting#paginate) here as well

Reply:
I don't understand how fronted would work differently if we change this

Reply:
You mean adding limit to the query?

Reply:
Because we don't have the same logic for calculating the offset. In Mongo, we use Pages and offset means the page number, not the index on the list as you have here. So if you can, please adjust it, because for Mongo it's not possible to do this in the way like you did

Reply:
Oh, I don't know if it is possible like that in postgres. We need to pass how many records we want to skip. But it could easily be calculated